igvm/vm.py

```diff
@@ -659,9 +659,9 @@ def aws_build(self,
         """
 
         vm_types_overview = self.aws_get_instances_overview()
-        if vm_types_overview:
```

Comment:
Seems aws_get_instances_overview can return None, and then there is iteration aws_get_fitting_vm_types through overview var which can be None and then probably lead to TypeError.